internal/tools/tools.go

```diff
@@ -16,6 +16,7 @@ import (
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/goware/modvendor"
 	_ "github.com/stormcat24/protodep"
+	_ "github.com/uber-go/gopatch"
```

Comment:
You need to add the `gopatach` to the list as well https://github.com/DataDog/datadog-agent/blob/494f98eb5db09652258c0e0aac936a35d69d7b62/tasks/install_tasks.py#L15-L16

Reply:
Good catch 👍